product_customer_code/models/product.py

```diff
@@ -0,0 +1,36 @@
+# coding: utf-8
+
+from openerp import api, fields, models
+
+
+class ProductProduct(models.Model):
+
+    _inherit = "product.product"
+
+    product_customer_code_ids = fields.One2many(
+        'product.customer.code', 'product_id', string='Product customer codes',
+        help='Customer Codes')
+
+    @api.multi
+    def copy(self, default=None):
```

Comment:
This change is in [5c0b939](https://github.com/Vauxoo/addons-vauxoo/pull/1206/commits/5c0b9397756425c1fdb19ec999454787a950e1fb)